legacy/reap/regionheap.h

```diff
@@ -47,7 +48,7 @@ using namespace HL;
 template <class Sbrk>
 class LeaHeap2 :
   public
-  Threshold<4096,
+  Threshold<HL::CPUINfo::PageSize,
```

Comment:
Typo here? (`CPUINfo` vs. `CPUInfo`)

Reply:
Thanks, I guess the template wasn't used in this codebase.